workspace-manager-clienttests/README.md

```diff
@@ -38,13 +38,13 @@ The version of the Workspace Manager client JAR file is specified in the build.g
 fetched from the Broad Institute Maven repository. You can override this to use a local version of the Workspace Manager client
 JAR file by specifying a Gradle project property, either with a command line argument
 
-`./gradlew -Pworkspacemanagerclientjar=~/terra-workspace-manager/workspace-manager-client/build/libs/workspace-manager-client-0.5.0-SNAPSHOT.jar runTest --args="configs/BasicUnauthenticated.json"
+`./gradlew -Pworkspacemanagerclientjar=~/terra-workspace-manager/workspace-manager-client/build/libs/workspace-manager-client-0.5.0-SNAPSHOT.jar runTest --args="integration/BasicAuthenticated.json"
 
 or an environment variable.
 
 ```
 export ORG_GRADLE_PROJECT_workspacemanagerclientjar=~/terra-workspace-manager/workspace-manager-client/build/libs/workspace-manager-client-0.5.0-SNAPSHOT.jar
-./gradlew runTest --args="configs/BasicUnauthenticated.json /tmp/TestRunnerResults"
+./gradlew runTest --args="integration/BasicAuthenticated.json /tmp/TestRunnerResults"
```

Comment:
So should we rename the directory from `-clienttests` to `-integrationtests`? I'm confused as to the degree these terms are interchangeable.

Reply:
The reference to `client` presents the ability of reusing the published or local client jar for writing test cases. Whether it is `smoke`, `integration`, or `perf` test, developers should be able to write the test using the same test runner framework. The client jar we are leveraging here is defined in the corresponding `build.gradle`. 

```
    // Workspace Manager client library
    // Gradle project property "workspacemanagerclientjar" overrides the fetch from Maven
    if (project.hasProperty("workspacemanagerclientjar")) {
        implementation files(project.findProperty("workspacemanagerclientjar"))
        compile "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jackson}"
        compile "org.glassfish.jersey.core:jersey-client:${jersey}"
        compile "org.glassfish.jersey.media:jersey-media-json-jackson:${jersey}"
        compile "org.glassfish.jersey.media:jersey-media-multipart:${jersey}"
        compile "org.glassfish.jersey.inject:jersey-hk2:${jersey}"
    } else {
        implementation "bio.terra:workspace-manager-client:${workspacemanagerClient}"
        compile "io.swagger.core.v3:swagger-annotations:${swaggerAnnotations}"
    }
```